h/models/group.py

```diff
@@ -33,6 +34,15 @@ class WriteableBy(enum.Enum):
     members = "members"
 
 
+class GroupMembershipRoles(enum.StrEnum):
+    """The valid role strings that're allowed in the GroupMembership.roles column."""
+
+    MEMBER = "member"
+    MODERATOR = "moderator"
+    ADMIN = "admin"
+    OWNER = "owner"
```

Comment:
Pulled these four strings out into an enum because they'll inevitably need to be referenced from elsewhere in the code later